packages/api/src/providers/index.ts

```diff
@@ -40,13 +42,26 @@ function accountToProvider<
   providerMap: TProviderMap,
 ): TProvider {
   if (!activeAccount.accessToken || !activeAccount.refreshToken) {
-    throw new Error("Invalid account");
+    const missingToken = !activeAccount.accessToken
+      ? "access token"
+      : "refresh token";
+    const errorMessage = `Invalid account: Missing ${missingToken} for provider '${activeAccount.providerId}' (accountId: ${activeAccount.accountId})`;
+    console.error(errorMessage);
+    throw new TRPCError({
+      code: "BAD_REQUEST",
+      message: errorMessage,
+    });
   }
```

Comment:
split this into separate checks, don't use console here, we already print it elsewhere